prims_algorithm.cpp

```diff
@@ -44,3 +55,12 @@ int main()
 }
 
 
+/*
```

Comment:
Remove this */ one is enough

Reply:
this one is for sample output description

Reply:
/*
Sample Output:
Edge  Value
1 - 0   2
2 - 1   3
3 - 0   6
4 - 1   5
*/

Reply:
push the input output on top

Reply:
Done